Source/DataSources/KmlDataSource.js

```diff
@@ -3349,6 +3349,7 @@ function load(dataSource, entityCollection, data, options) {
  * @property {Boolean} [clampToGround=false] true if we want the geometry features (Polygons, LineStrings and LinearRings) clamped to the ground.
  * @property {Ellipsoid} [ellipsoid=Ellipsoid.WGS84] The global ellipsoid used for geographical calculations.
  * @property {Credit|String} [credit] A credit for the data source, which is displayed on the canvas.
+ * @property {KmlTour} [KmlTours] The KmlTour that is used to guide the camera to specified destinations on given time intervals.
```

Comment:
This isn't the correct place to put the documentation since `kmlTours` is not a property of the constructor options but rather a property of the data source itself. The fix will require a few changes:

* Add a private member variable to `KmlDataSource`. In the constructor right underneath `this._resourceCredits` add `this._kmlTours = [];`
* Add a `kmlTours` getter to `Object.defineProperties`. No need to add a setter. It will will look something like:

```js
  /**
   * Gets the KML Tours that are used to guide the camera to specified destinations on given time intervals.
   * @memberof KmlDataSource.prototype
   * @type {KmlTour[]}
   */
  credit: {
    get: function () {
      return this._credit;
    },
  }
```
* Any code inside `KmlDataSource` should use `_kmlTours`; anything outside should use `kmlTours`. For example, the Sandcastle will continue to use `kmlTours`.
* The following lines can be removed because the empty array will be initialized in the constructor
```js
if (!defined(dataSource.kmlTours)) {
    dataSource.kmlTours = [];
}
```

Reply:
@lilleyse thank you for the feedback. I just added a new commit that should address your suggestions.